exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/DDLTests.kt

```diff
@@ -1124,19 +1123,6 @@ class DDLTests : DatabaseTestsBase() {
         }
     }
 
-    internal enum class Foo {
-        Bar, Baz;
-
-        override fun toString(): String = "Foo Enum ToString: $name"
-    }
-
-    class PGEnum<T : Enum<T>>(enumTypeName: String, enumValue: T?) : PGobject() {
-        init {
-            value = enumValue?.name
-            type = enumTypeName
-        }
-    }
-
```

Comment:
These are only used in `EnumerationTests.kt`.
When that new file was created in [commit c897fdb](https://github.com/JetBrains/Exposed/commit/c897fdb455c76a442ed4dee65996b876507d6609#diff-422f0177ef1eb040e04af1efc6e7494f30168f15e26f50b613de592b62c4a3f5), they just weren't carried over too.